easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.15.1-foss-2023a.eb

```diff
@@ -0,0 +1,246 @@
+easyblock = 'PythonBundle'
+
+name = 'TensorFlow'
+version = '2.15.1'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'pic': True}
+
+use_pip = True
+sanity_pip_check = True
+
+
+builddependencies = [
+    ('Bazel', '6.1.0'),
+    ('git', '2.41.0', '-nodocs'),
+    ('pybind11', '2.11.1'),
+    ('UnZip', '6.0'),
+    ('poetry', '1.5.1'),
+    ('Clang', '16.0.6'),
+    ('LLVM', '16.0.6'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('h5py', '3.9.0'),
+    ('cURL', '8.0.1'),
+    ('dill', '0.3.7'),
+    ('double-conversion', '3.3.0'),
+    ('flatbuffers', '23.5.26'),
+    ('flatbuffers-python', '23.5.26'),
+    ('giflib', '5.2.1'),
+    ('hwloc', '2.9.1'),
+    ('ICU', '73.2'),
+    ('JsonCpp', '1.9.5'),
+    ('libjpeg-turbo', '2.1.5.1'),
+    ('NASM', '2.16.01'),
+    ('nsync', '1.26.0'),
+    ('SQLite', '3.42.0'),
+    ('patchelf', '0.18.0'),
+    ('protobuf-python', '4.23.4'),
+    ('libpng', '1.6.39'),
+    ('snappy', '1.1.10'),
+    ('zlib', '1.2.13'),
+    ('RE2', '2023-08-01'),
+    ('OpenSSL', '1.1', '', SYSTEM),
+]
+
+exts_list = [
```

Comment:
Out of curiousity: how did you get to the list of dependencies? Did you use the `findPythonDeps.sh` script from EasBuild? https://docs.easybuild.io/api/easybuild/scripts/findPythonDeps/

Reply:
I think yes, but actuall it was taken from @Flamefire  's [gist](https://gist.github.com/Flamefire/49426e502cd8983757bd01a08a10ae0d). But I've tweaked versions so stripped the comment away, will work on it along with the other comments.

Reply:
Thanks for adding the comments, very clear now :)